Views/GIConflictResolverViewController.m

```diff
@@ -61,8 +61,8 @@ - (void)viewWillShow {
   XLOG_DEBUG_CHECK(self.repository.statusMode == kGCLiveRepositoryStatusMode_Disabled);
   self.repository.statusMode = kGCLiveRepositoryStatusMode_Unified;
   
-  _oursTextField.stringValue = [NSString stringWithFormat:@"\"%@\" <%@>", _ourCommit.summary, _ourCommit.shortSHA1];
```

Comment:
This should be changed into NSLocalizedString() in a separate and prior commit to make it clear it's a UI string.